repro/run_on_host.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -eux
+
+cd $(dirname ${BASH_SOURCE})
+
+if [[ ! -f ./repro.sif ]]; then
+    singularity build --fakeroot repro.sif repro.def
+fi
+
+mkdir -p bazel_cache
+
+if [[ ! -f ./overlay.img ]]; then
+    dd if=/dev/zero of=overlay.img bs=1M count=500
+    mkfs.ext4 overlay.img
+fi
+
+# TODO(eric): How to make sandbox persistent?
```

Comment:
```
apptainer build --sandbox --fakeroot repro_sandbox repro.def
```

Would build a persistent sandbox instead of a `sif`.

Reply:
BTW Oh... didn't realize that `singularity` just got EOL'ed... But thanks! Will try out newer setup.

Reply:
Done (messily) in 4b940f3.

Am now able to repro, but that is actually quite concerning.

Reply:
Ah, retrying c8c1867 with a fresh setup (no bazel cache, rebuild image) is able to reproduce it now. I'm guessing Bazel had a false cache hit (`--nv` made changes that our repository rule isn't sensitive to).